packages/cli/src/commands/push-browser-destinations.ts

```diff
@@ -21,12 +21,17 @@ export default class PushBrowserDestinations extends Command {
   static examples = [`$ ./bin/run push-browser-destinations`]
 
   static flags = {
-    help: flags.help({ char: 'h' })
+    help: flags.help({ char: 'h' }),
+    env: flags.string({
+      char: 'e',
+      default: 'stage'
+    })
```

Comment:
You can source this from env vars too: 
```suggestion
    env: flags.string({
      char: 'e',
      default: 'stage',
      env: 'NODE_ENV' // or whatever
    })
```

Reply:
Also worth adding a description to the flag!